certified-connectors/AIForged/apiProperties.json

```diff
@@ -0,0 +1,66 @@
+{
+  "publisher": "Larc AI (PTY) Ltd",
+  "stackOwner": "Larc AI (PTY) Ltd",
+  "contact": {
+    "name": "AIForged",
+    "url": "https://www.aiforged.com",
+    "email": "info@aiforged.com"
+  },
+  "iconBrandColor": "#0E76BC",
+  "capabilities": [
+    "actions"
+  ],
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "AIForged",
+          "clientId": "swaggerui",
+          "clientSecret": "read the docs",
+          "scopes": [
+            "openid",
+            "email",
+            "phone",
+            "profile",
+            "roles",
+            "aiforgedapi"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False"
+          }
+        },
+        "uiDefinition": {
+          "displayName": "Log in with AIForged Account",
+          "description": "Log in with AIForged Account",
+          "tooltip": "Log in with AIForged Account",
+          "constraints": {
+            "required": "true"
+          }
+        },
+        "customParameters": {
+          "discoveryUrl": {
+            "value": "https://portal.aiforged.com/.well-known/openid-configuration"
+          },
+          "authorizationUrl": {
+            "value": "https://portal.aiforged.com/connect/authorize"
+          },
+          "tokenUrl": {
+            "value": "https://portal.aiforged.com/connect/token"
+          }
+        },
+        "identityProvider": "oauth2",
```

Comment:
@aiforged The "identityProvider" property is already mentioned above and it was incorrect. Please remove it.